Automattic/jetpack
# Protect: Configure Sync to minimal required data

#### Changes proposed in this Pull Request:

* Configure Sync to only sync what's required for Protect to work

#### Jetpack product discussion


p1HpG7-fgl-p2

#### Does this pull request change what data or activity we track or use?


It reduces the amount of data synced. But since this plugin was not released yet, there's no change to anything at this point.

#### Testing instructions:






* Enable only Jetpack Protect Plugin
* Enable Jetpack Debug helper plugin
* Connect
* Go to Jetpack Debug > Sync Data Settings Tester
* Confirm you see a small list of thing. 3 items under jetpack_sync_options_whitelist (including `active_plugins`) and 5 items under jetpack_sync_callable_whitelist including `get_plugins`
